frontend/src/app/edit-claim/edit-claim.component.ts

```diff
@@ -121,7 +121,17 @@ export class EditClaimComponent implements OnInit {
   }
 
   downloadFile(hash: string, filename: string) {
-    this.service.downloadFile(this.vin, hash, filename);
+    this.service.downloadFile(this.vin, hash, filename).subscribe(blob => {
+      const url = URL.createObjectURL(new File([blob], filename));
+      const a = document.createElement('a');
+      a.href = url;
+      a.target = '_blank';
+      a.download = filename;
+      a.style.display = 'none';
+      document.body.appendChild(a);
+      a.click();
+      document.body.removeChild(a);
```

Comment:
This seems like a lot for a download prompt - but I've also not done much with downloads so this is fine.

Reply:
You can thank angular for being shitty for that.